framework/packages/abstract-interface/src/ibc.rs

```diff
@@ -59,14 +59,33 @@ impl<Chain: CwEnv> AbstractIbc<Chain> {
             ),
         ])
     }
+
+    #[cfg(feature = "interchain")]
+    /// Register infrastructure on connected chains
+    ///
+    /// Note: this should only be called on real chains (for example using [`cw_orch::daemon::Daemon`])
```

Comment:
If so, should it not be implemented for `Abstract<Daemon>` instead of `Abstract<CwEnv>`?